src/legacy/core_plugins/kibana/public/management/sections/settings/__snapshots__/advanced_settings.test.tsx.snap

```diff
@@ -322,7 +322,7 @@ exports[`AdvancedSettings should render specific setting if given setting key 1`
     }
     showNoResultsMessage={true}
   />
-  <advanced_settings_page_footer
+  <[object Object]
```

Comment:
Were these snapshots supposed to change at all? `[object Object]` feels suspicious...

Reply:
There was code that set the `displayName` for the components in the registry which I removed. It was giving me trouble, its only used for debugging (and snapshots I guess), and this code is used in one particular place. I feel comfortable leaving this as it currently is but could dig deeper if prodded.

Reply:
I should probably dig into this a little more

Reply:
Ug, I'm having trouble finding a good solution. Its reasonable that it changes but its changing to something meaningless. Any advice?

Reply:
> There was code that set the displayName for the components in the registry which I removed.

Ahh that makes sense then. Looks like if you don't explicitly set a `displayName` in React, [it will just try to infer a name](https://reactjs.org/docs/react-component.html#displayname). So my assumption is that React is trying/failing to infer a name for some reason. Do you think it's worth keeping `displayName` in there for debugging purposes? Curious what troubles it was causing.